step-templates/octopus-register-listening-target.json

```diff
@@ -1,135 +1,135 @@
-        }
-      },
-      {
-        "Id": "667a18b3-1694-4333-87a4-9be712b59122",
-        "Name": "RegisterListeningTarget.Tenant.List",
-        "Label": "Tenant CSV List",
-        "HelpText": "(Optional) If this is for a tenant, the a comma separated list of tenants to assign the machine to in Octopus Deploy",
-        "DefaultValue": "",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "db2de612-45bb-4e4a-ab95-64083dd80393",
-        "Name": "RegisterListeningTarget.Tenant.DeploymentType",
-        "Label": "Tenanted Deployments",
-        "HelpText": "Choose the kind of deployment where this deployment target should be included.",
-        "DefaultValue": "Untenanted",
-        "DisplaySettings": {
-          "Octopus.ControlType": "Select",
-          "Octopus.SelectOptions": "Untenanted|Exclude from tenanted deployments (default)\nTenanted|Include only in tenanted deployments\nTenantedOrUntenanted|Include in both tenanted and untenanted deployments"
-        }
-      },
-      {
-        "Id": "5ef91f37-b13b-413d-955c-872c7f274c7e",
-        "Name": "RegisterListeningTarget.MachinePolicy.IdOrName",
-        "Label": "Machine Policy Id Or Name",
-        "HelpText": "Enter in the name or the Id of the Machine Policy in Octopus Deploy for the AKS Cluster.",
-        "DefaultValue": "Default Machine Policy",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "5b2e0993-7f96-4447-b47b-029c8f225688",
-        "Name": "RegisterListeningTarget.Overwrite.Existing",
-        "Label": "Overwrite Existing Registration",
-        "HelpText": "Indicates if the existing listening tentacle should be overwritten in Octopus.",
-        "DefaultValue": "False",
-        "DisplaySettings": {
-          "Octopus.ControlType": "Checkbox"
-        }
+    {
+      "Id": "25dd9d35-448e-4536-b622-d8d2fce8d4f4",
+      "Name": "RegisterListeningTarget.Roles.List",
+      "Label": "Role CSV List",
+      "HelpText": "Comma separated list of environments to assign to the machine in Octopus Deploy.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
       }
-    ],
-    "LastModifiedBy": "octopusbob",
-    "$Meta": {
-      "ExportedAt": "2020-04-13T15:37:29.866Z",
-      "OctopusVersion": "2020.1.10",
-      "Type": "ActionTemplate"
     },
-    "Category": "octopus"
-  }
\ No newline at end of file
+    {
+      "Id": "62c64e03-7fec-4cdb-8a9c-4038f767ffd2",
+      "Name": "RegisterListeningTarget.Environment.List",
+      "Label": "Environment CSV List",
+      "HelpText": "Comma separated list of environments to assign to the the machine in Octopus Deploy.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "8d80c23a-7f5d-43c4-885b-c6696b947d20",
```

Comment:
This looks like a new ID, did you paste in the entire JSON body or just the script?